example/example.ml

```diff
@@ -56,41 +55,48 @@ module MyAPI(R : RPC) = struct
 
 end
 
+module IdM = struct
```

Comment:
In the next rewrite, in preparation, I am going to make this into a real exception monad with `type 'a t = Value 'a | Exception exn`, and make the tests consistent